Ravel-Creators/Assets/Ravel-Creators/Scripts/Editor/RavelEditor.cs

```diff
@@ -1,15 +1,27 @@
+using System;
 using Base.Ravel.Networking.Authorization;
+using Base.Ravel.Networking.Organisations;
 using Base.Ravel.Users;
 using UnityEditor;
 using UnityEngine;
 
+/// <summary>
+/// Manager class for editor user data and usefull editor functions
+/// </summary>
 public static class RavelEditor
 {
     public static bool LoggedIn {
         get { return User != null; }
     }
     public static  User User { get; set; }
-
+    public static bool DevUser { get; private set; }
+    
+    private static bool _retrievingOrganisations;
+    private static Action<Organisation[], bool> _onOrgsRetrieved;
+    
+    /// <summary>
+    /// This object contains custom settable branding files for the editor tools.
+    /// </summary>
     public static RavelBranding Branding {
         get {
             if (_branding == null) {
```

Comment:
Throw error when there is no config

Reply:
Error staat er in, maar de bedoeling is dat dit onze file blijft en niet degene van users denk ik. Maarja better be sure